dev/MRTCore/.gitignore

```diff
@@ -0,0 +1,341 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
There's already a .gitignore in the root of the tree. Consider merging any missing content from here into there instead.

Reply:
it's not clear yet how independent this submodule will be. For now I think it's good to leave it self contained.

---
In reply to: [488400054](https://github.com/microsoft/ProjectReunion/pull/185#discussion_r488400054) [](ancestors = 488400054)

Reply:
This is not a submodule - it's part of the PR repo. We can discuss in a further change.

Reply:
not a submodule, but a child project. once it's settled, we will make changes to blend in.

---
In reply to: [488957827](https://github.com/microsoft/ProjectReunion/pull/185#discussion_r488957827) [](ancestors = 488957827)

Reply:
+1. this is the "no friction" onboarding, we will do an "integration" change soon-ish.